Source/Scene/createElevationBandMaterial.js

```diff
@@ -0,0 +1,572 @@
+import Cartesian4 from "../Core/Cartesian4.js";
+import CesiumMath from "../Core/Math.js";
+import Check from "../Core/Check.js";
+import Color from "../Core/Color.js";
+import defaultValue from "../Core/defaultValue.js";
+import defined from "../Core/defined.js";
+import DeveloperError from "../Core/DeveloperError.js";
+import mergeSort from "../Core/mergeSort.js";
+import PixelFormat from "../Core/PixelFormat.js";
+import PixelDatatype from "../Renderer/PixelDatatype.js";
+import Sampler from "../Renderer/Sampler.js";
+import Texture from "../Renderer/Texture.js";
+import TextureMagnificationFilter from "../Renderer/TextureMagnificationFilter.js";
+import TextureMinificationFilter from "../Renderer/TextureMinificationFilter.js";
+import TextureWrap from "../Renderer/TextureWrap.js";
+import Material from "./Material.js";
+
+var scratchColor = new Color();
+var scratchColorAbove = new Color();
+var scratchColorBelow = new Color();
+var scratchColorBlend = new Color();
+var scratchPackedFloat = new Cartesian4();
+var scratchColorBytes = new Uint8Array(4);
+var blankColor = new Color(0.0, 0.0, 0.0, 0.0);
```

Comment:
This could be Color.BLACK ?

Reply:
It needs to be all zeros. `console.log(Cesium.Color.BLACK);` -> `(0, 0, 0, 1)`